### PR TITLE
removes the test-merge only code from the combat mode commit

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
fix: removes the test-merge only code from the combat mode commit
/:cl:

### CODE REVIEW

File: code/game/objects/items/melee/misc.dm [hunk 1]
```diff
@@ -173,7 +173,7 @@
 
 /obj/item/melee/classic_baton
 	name = "police baton"
-	desc = "A wooden truncheon for beating criminal scum."
+	desc = "A wooden truncheon for beating criminal scum. Left click to stun, right click to harm"
```
Comment (Jared-Fogle) [inline code/game/objects/items/melee/misc.dm:176]:
```suggestion
	desc = "A wooden truncheon for beating criminal scum. Left click to stun, right click to harm."
```

File: code/game/objects/items/stunbaton.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/item/melee/baton
 	name = "stun baton"
-	desc = "A stun baton for incapacitating people with. Left click to harm, right click to stun."
+	desc = "A stun baton for incapacitating people with. Left click to stin, right click to harm."
```
Comment (Jared-Fogle) [inline code/game/objects/items/stunbaton.dm:3]:
```suggestion
	desc = "A stun baton for incapacitating people with. Left click to stun, right click to harm."
```

Comment (Cyberboss) [inline code/game/objects/items/stunbaton.dm]: ^

Comment (bobbahbrown) [inline code/game/objects/items/stunbaton.dm]: stin

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Kylerace** [comment]
lol you really forgot to take out the testmerge stuff before it got merged? smh

File: code/modules/mob/dead/new_player/login.dm [hunk 3]
```diff
@@ -21,7 +21,6 @@
 		to_chat(src, "<span class='notice'><b>Admin Notice:</b>\n \t [GLOB.admin_notice]</span>")
 
 	to_chat(src, "<span class='notice'><b>Combat Mode Changes</b>\n \t</span>")
```
Comment (bobbahbrown) [inline code/modules/mob/dead/new_player/login.dm:23]: you left this line in

**Review by bobbah — CHANGES_REQUESTED** [review]

**Pull request merged**